### DISCUSSION TITLE
Raw tablet input in relative mode

### DISCUSSION THREAD

**Discussion by 4310V343k** [ppy/osu#17757] [Q&A] [answered]
How can i use the relative mode while using raw input on Linux?

**Comment by 4310V343k** [comment, marked as answer]
After some searching found this
[ppy/osu-framework@`1a87d37` (#4162)](https://github.com/ppy/osu-framework/pull/4162/commits/1a87d3710baa28b439761450226f1e533508a7ff)
Looks like this was removed on purpose

**Reply by peppy** [reply]
Stable does not have relative mode either. I'm not sure what you're talking about. Maybe try turning off the tablet support in lazer and see if that gives you what you want?

**Reply by 4310V343k** [reply]
Stable doesn't seem to have any raw input mode for tablets at all
Lazer has it, but only for absolute input

Of course, if i disable it the tablet will use system-wide OTD or any other driver, and in that driver i'll be able to use the relative mode

**Reply by peppy** [reply]
Again, you can turn it off in the settings. Which will give you the same thing that you are used to, I guess. Search for "tablet" and click to disable it.
